langlib/lang.map/src/main/ballerina/map.bal

```diff
@@ -59,6 +71,12 @@ public isolated function iterator(map<Type> m) returns object {
 # This for use in a case where it is known that the map has a specific key,
 # and accordingly panics if parameter `m` does not have a member with parameter `k` key.
 #
+# ```ballerina
+# {"Carl": 85, "Bob": 50, "Max": 60}.get("Carl") ⇒ 85
+#
+# {"Carl": 85, "Bob": 50, "Max": 60}.get("John") ⇒ panic
```

Comment:
Since we are using the same mapping value in two calls we can introduce a variable here.